README.md

```diff
@@ -17,41 +18,56 @@ These instructions will provide the information you need to extract data from DC
 * Python 3.8+: [Python Install](https://www.python.org/downloads)
```

Comment:
Probably remove the v3.8 piece here if it's repeated below.

Reply:
Kept it here and removed the below line